map_path: "$(find top_down_render)/maps/grace_quarters"
use_raster: False
svg_res: 3.06
svg_origin_x: 0
svg_origin_y: 0
init_pos_px_x: 644
init_pos_px_y: 654
init_pos_px_cov: 5
